### PR TITLE
Add devise

### PR DESCRIPTION
**Pull request by christianbiring1**
In this project, I had to add the [devise](https://github.com/heartcombo/devise) gem to my app and use it for the registration and login of users.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (BenMukebo) [inline app/models/user.rb]: - [x] Great job so far 👏🏻  However according to the project requirements, the application should ask for confirmation of email. Let's enabled the user `:confirmable` in the user model. You should see something similar to what is below:
    ```ruby
    	devise :database_authenticatable, :registerable,
    		    :recoverable, :rememberable, :validatable, :confirmable
    ````

- [x]  Kindly, let's make sure the user can reset the password in the application. Let's enabled the user  `:recoverable`  in the user model. You should see something similar to what is below:

   ```ruby
   	devise :database_authenticatable, :registerable,
   		    :recoverable, :rememberable, :validatable, :confirmable
   ```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @christianbiring1,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️    
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (christianbiring1) [reply app/models/user.rb]: Thank you so much @BenMukebo , let me fix that real quick :+1:  :rocket:

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @christianbiring1, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**